uportal-war/src/main/java/org/apereo/portal/security/provider/AnyUnblockedGrantPermissionPolicy.java

```diff
@@ -155,7 +155,7 @@ public boolean doesPrincipalHavePermission(
         boolean rslt;
         try {
             // Track groups we've already explored to avoid infinite loop
-            final Set<IGroupMember> seenGroups = new HashSet<IGroupMember>(100);
+            final Set<IGroupMember> seenGroups = new HashSet<>(100);
```

Comment:
Question purely from curiosity, why start the initial capacity of the set to 100?